stub-connector/src/main/java/uk/gov/ida/notification/stubconnector/resources/ReceiveResponseResource.java

```diff
@@ -77,21 +82,22 @@ public ResponseView receiveResponse(
 
         // The eIDAS Response should only contain one Assertion with one AttributeStatement which contains
         // the user's requested attributes
-
-        List<String> attributes = new ArrayList<>();
-
         Response decrypted = decrypter.decrypt(response);
         List<Assertion> assertions = decrypted.getAssertions();
 
+        List<Entry<String, String>> attributesByName = new ArrayList<>();
```

Comment:
Any reason this isn't a Map?

Reply:
Because Map orders the attributes in alphabetical order by name and then it looks weird onn the result page :trollface:

Reply:
Ok, could perhaps use LinkedHashMap but it works as is!